new-lamassu-admin/src/pages/Wallet/Wallet.js

```diff
@@ -99,47 +103,60 @@ const Wallet = ({ name: SCREEN_KEY }) => {
 
   return (
     <>
-      <TitleSection title="Wallet Settings" />
-      <EditableTable
-        name="test"
-        namespaces={R.map(R.path(['code']))(cryptoCurrencies)}
-        data={config}
-        error={error?.message}
-        stripeWhen={it => !WalletSchema.isValidSync(it)}
-        enableEdit
-        shouldOverrideEdit={shouldOverrideEdit}
-        editOverride={setWizard}
-        editWidth={174}
-        save={save}
-        validationSchema={WalletSchema}
-        elements={getElements(cryptoCurrencies, accountsConfig, onChange)}
+      <TitleSection
+        title="Wallet Settings"
+        button={{
+          text: 'Advanced settings',
+          icon: SettingsIcon,
+          inverseIcon: ReverseSettingsIcon,
+          toggle: setAdvancedSettings
+        }}
       />
-      {wizard && (
-        <Wizard
-          coin={R.find(R.propEq('code', wizard))(cryptoCurrencies)}
-          onClose={() => setWizard(false)}
-          save={save}
-          error={error?.message}
-          cryptoCurrencies={cryptoCurrencies}
-          userAccounts={data?.config?.accounts}
-          accounts={accounts}
-          accountsConfig={accountsConfig}
-        />
-      )}
-      {editingSchema && (
-        <Modal
-          title={`Edit ${editingSchema.name}`}
-          width={478}
-          handleClose={() => setEditingSchema(null)}
-          open={true}>
-          <FormRenderer
-            save={wizardSave}
-            elements={editingSchema.elements}
-            validationSchema={editingSchema.validationSchema}
-            value={accounts[editingSchema.code]}
+      {!advancedSettings && (
+        <>
+          <EditableTable
+            name="test"
+            namespaces={R.map(R.path(['code']))(cryptoCurrencies)}
+            data={config}
+            error={error?.message}
+            stripeWhen={it => !WalletSchema.isValidSync(it)}
+            enableEdit
+            shouldOverrideEdit={shouldOverrideEdit}
+            editOverride={setWizard}
+            editWidth={174}
+            save={save}
+            validationSchema={WalletSchema}
+            elements={getElements(cryptoCurrencies, accountsConfig, onChange)}
           />
-        </Modal>
+          {wizard && (
+            <Wizard
+              coin={R.find(R.propEq('code', wizard))(cryptoCurrencies)}
+              onClose={() => setWizard(false)}
+              save={save}
+              error={error?.message}
+              cryptoCurrencies={cryptoCurrencies}
+              userAccounts={data?.config?.accounts}
+              accounts={accounts}
+              accountsConfig={accountsConfig}
+            />
```

Comment:
We don't want to create a migration every time we add a coin. We either add the option of display units on the wizard, or grab the first available unit on a coin and save automatically on first config.

Reply:
I think I'm not following, this code didn't change